packages/components/alert/style/index.less

```
@@ -46,24 +46,30 @@
   }
 
   &-pagination {
-    margin-left: @alert-action-margin-left;
-    line-height: @alert-icon-font-size;
-    color: @alert-color-secondary;
-
-    &-text {
-      padding: @alert-pagination-text-padding;
-    }
-
-    &-icon {
-      cursor: pointer;
-      vertical-align: middle;
-      &:hover {
-        color: @alert-color;
-      }
-
-      &.@{alert-prefix}-pagination-disabled {
-        cursor: not-allowed;
-        color: @alert-disabled-color;
+    &.@{pagination-prefix} {
+      margin-left: @alert-action-margin-left;
+      color: @alert-color-secondary;
+
+      .@{pagination-prefix} {
+        &-item-slash {
+          margin: @alert-pagination-text-margin;
+        }
+
+        &-item {
+          margin-right: 0;
+
+          &-button {
+            .@{button-prefix}:not(.@{button-prefix}-disabled):hover {
+              background-color: transparent;
+            }
+
+            .@{button-prefix}:not(.@{button-prefix}-disabled):not(.@{button-prefix}-danger):not(.@{button-prefix}-primary):not(.@{button-prefix}-link):not(:hover):not(:focus):not(:active) {
```

```suggestion
            .@{button-prefix}-default:not(.@{button-prefix}-disabled):not(:hover):not(:focus):not(:active) {
```
给 按钮默认情况下加个 default 的 class ? 很早之前就想这么干了来着。